src/commands/paint.ts

```diff
@@ -213,22 +215,26 @@ export function paint(
   }
 
   bus.on('WORKER_MSG', ({id, msg}) => {
+    setupWorker(id);
```

Comment:
Like we talked about, this is a dumb function that lets you emit a message to a dynamically-generated paint row (added so plugins can get their own row, which we don’t always know about until things build). This can be removed (not a focus).